source/util/AMRecursiveDirectoryCompare.cpp

```diff
@@ -215,17 +215,22 @@ bool AMRecursiveDirectoryCompare::compareOneLevel(const QString &path1, const QS
 	}
 	onlySide1SubDirectories = side1SubDirectories;
 
+
+	if (unknownFiles.count() > 0)
+		qDebug() << QString("Found (%1) files for which the more up-to-date version cannot be determined, leading with %2.").arg(unknownFiles.count()).arg(unknownFiles.at(0));
 	for (int x = 0, size = unknownFiles.count(); x < size; x++) {
-		qDebug() << "Found files for which the more up-to-date version cannot be determined";
 		unknownFiles_.append(QDir::cleanPath(QString(".../%1").arg(unknownFiles.at(x))));
 	}
 
-	for(int x = 0, size = newerSide1Files.count(); x < size; x++){
+	if (newerSide1Files.count() > 0)
```

Comment:
Look at the original codes, the qDebug() is sitting inside the loop. That is, we will print out the same message `count()` times, which only gives us a single information.

The new changes will only qDebug() a single line.